src/components/Button/index.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+type buttonProps = {
```

Comment:
Type name should be PascalCase.
```suggestion
type ButtonProps = {
```